.github/workflows/build-apps.yml

```diff
@@ -233,14 +226,7 @@ jobs:
           CSC_KEY_PASSWORD: ${{ secrets.APPLE_CERTIFICATE_PASSWORD }}
           CSC_KEYCHAIN: ${{ secrets.APPLE_SIGNING_IDENTITY }}
           WINDOWS_CERTIFICATE_THUMBPRINT: ${{ secrets.WINDOWS_CERTIFICATE_THUMBPRINT }}
-          DEBUG: "electron-notarize*"
-        # TODO: Fix electron-notarize flakes. The logs above should help gather more data on failures
-        uses: nick-fields/retry@v3.0.2
-        with:
-          shell: bash
-          timeout_minutes: 10
-          max_attempts: 3
-          command: npm run tronb:package:prod
+        run: npm run tronb:package:prod
```

Comment:
Taking the chance to take this off as well, we haven't seen notarization flakes in a while